tests/VerificationClassTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+// PHPUNITが実行される場所がカレントディレクトリになるので要注意
+require_once './Verification/VerificationClass.php';
+
+class VerificationClassTest extends PHPUnit_Framework_TestCase {
+
+	// TEST対象となるクラス
+	protected $VerificationClass = '';
+
+	// 初期設定でクラスをインスタンス化
+	public function setUp() {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening brace should be on a new line